packages/server/src/login/login.controller.ts

```diff
@@ -143,6 +148,11 @@ export class LoginController {
     @Query('userId') userId: number,
   ): Promise<void> {
     const token = await this.jwtService.signAsync({ userId });
-    res.cookie('auth_token', token).redirect(302, '/');
+    const isSecure = this.configService
+      .get<string>('DOMAIN')
+      .startsWith('https://');
+    res
+      .cookie('auth_token', token, { httpOnly: true, secure: isSecure })
```

Comment:
Snuck this in: the `secure` attributes means this cookie can only be sent over https.